start.sh

```diff
@@ -1,12 +1,18 @@
-set -e 
+set -e
 
 if [ ${RAILS_ENV:-development} != "production" ]; then
   bundle check || bundle
+  yarn
```

Comment:
@whereismyjetpack this doesn't seem related to sidekiq. Is it?

Reply:
removed.

Reply:
@whereismyjetpack cool. Does it belong someplace else? I don't want to loose track of it if it is... we can ticket it or something if needed.

Reply:
I'll submit it as another PR!